src/include/common/types/uuid.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "common/types/int128_t.h"
+#include "common/types/ku_string.h"
+namespace kuzu {
+namespace common {
+
+struct uuid_t {
+    int128_t value;
+};
+
+struct UUID {
+    constexpr static const uint8_t STRING_SIZE = 36;
```

Comment:
remove const keyword as you have already declared it as constexpr. Better to name it UUID_STRING_LENGTH